sdk/go/common/apitype/updates.go

```diff
@@ -70,6 +70,13 @@ type UpdateMetadata struct {
 	Environment map[string]string `json:"environment"`
 }
 
+// Message is a message from the backend to be displayed to the user.
+type Message struct {
+	Message       string `json:"message"`
```

Comment:
I decided that this should be a struct to make it easier to extend the formatting and display capabilities into the future if requested.